README.md

```diff
@@ -26,6 +26,13 @@ pip install aixplain
 Before you can use the aixplain SDK, you'll need to obtain an API key from our platform. For details refer this [Team API Key Guide](docs/user/api_setup.md).
 
 Once you get the API key, you'll  need to add this API key as an environment variable on your system.
+
+#### Python (recommended)
+```python
+import os
+os.environ["TEAM_API_KEY"] = "YOUR_API_KEY"
+```
+
```

Comment:
@thiago-aixplain please change this to magic env for jupyter as discussed. Please do not mention that it is recommended since our users can use it from the terminal or any other software.